packages/user/TokenStream/plugin/wit/world.wit

```diff
@@ -3,9 +3,37 @@ package token-stream:plugin;
 interface api {
     use host:types/types.{error};
 
+
+    /// Creates a new token stream
+    ///
+    /// Parameters:
+    /// * `half-life-seconds`: Half life of the vesting rate in seconds
```

Comment:
I wouldn't say that it's half life of the vesting _rate_. Even though derivatives of all orders have the same half life, it's not the normal way to think about it.